src/Behat/Gherkin/Lexer.php

```diff
@@ -502,16 +511,24 @@ protected function scanTableRow()
     protected function scanTags()
     {
         $line = $this->getTrimmedLine();
+
         if (!isset($line[0]) || '@' !== $line[0]) {
             return null;
         }
 
+        if(preg_match('/^(?<line>.*)\s+#.*$/', $line, $matches)) {
+            ['line' => $line] = $matches;
+            $this->consumeLineUntil(mb_strlen($line, 'utf-8'));
+        }
+        else {
```

Comment:
why don't you follow PSR-12 here ?

Reply:
I don't find it very valuable, but I'll change

Reply:
Hm maybe that sounded too rude. It's PSR-12 I dislike, your reviews are invaluable

Reply:
I mean, the existing code already follows PSR-12 (or maybe the older PSR-2, we probably don't have code relying on new PHP features that were encoded in PSR-12). This is about being consistent.